common/tools/dev-tool/src/util/printer.ts

```diff
@@ -4,13 +4,14 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
 
 import chalk from "chalk";
-import path from "path";
 
 const printModes = ["info", "warn", "error", "success", "debug"] as const;
 
 export type Fn<T = void> = (...values: any[]) => T;
 export type ModeMap<T> = { [k in typeof printModes[number]]: T };
 
+const DEV_TOOL_PATH = __dirname.split("dev-tool")[0];
```

Comment:
Won't this fail if cloned path happened to have "dev-tool"?

Reply:
Good point. `__dirname.split(path.sep + "dev-tool" + path.sep).slice(0, -1).join(path.sep + "dev-tool" + path.sep)` is probably more reliable as long as we take care not to have a folder named `dev-tool` anywhere under the dev-tool folder. The most robust solution is to recursively look for the `package.json`, but I was trying to avoid doing that in module initialization.